README.md

```diff
@@ -14,6 +14,20 @@ language file.
 If you have a solution that is more complex than a single `.v` file, make a subdir with
 your GitHub ID under the appropriate year/day subdir, and place all files there.
 
+## Input file format
+
+1, Input file should ends with file extension `.input`
+2, Input file should be directly copy-paste from AoC's example input with the following 
+standard:
+- - No whitespace trimming (see note 1)
+- - No additional empty line
+- - No personal-specific input allowed to be example input file
+3, If there are multiple example inputs for different parts in a day, suffix different 
+input file with `-partX` where X is part number, then follows other citerions
+4, Input file's name should relates to corresponding AoC problem, for example: AoC 2022
+Day 7 relates to unix system's file system cmd operations, therefore, you could probably
+name it with`filesystem.input` or `cmd
+
 Note: when you add .input files, please turn off the auto trimming
```

Comment:
```suggestion
Note 1: when you add .input files, please turn off the auto trimming
```
Since it is referred to earlier as "Note 1", it should be named the same here.